pybossa/view/projects.py

```diff
@@ -1227,6 +1222,14 @@ def task_presenter(short_name, task_id):
             markup = Markup('{{}} <a href="{}">{{}}</a>'.format(url))
             flash(markup.format(msg_1, msg_2), "warning")
 
+    scheduler = project.info.get('sched', "default")
+    if not (current_user.subadmin or current_user.admin or current_user.id in project.owners_ids):
+        # Users without admin privilege have to be on task_queue-cherry_pick or be locked to task to view
```

Comment:
The task queue cherry pick option is only for regular user, that is non admin or non subadmin who's not co-owner to the project, whereas the above condition seems disallowing any subadmin or any co-owners. IMO, the condition could be 
```python
if not current_user.admin or not (current_user.subadmin and current_user.id in project.owners_ids):
```
which can also be written as
```python
if not (current_user.admin and (current_user.subadmin and current_user.id in project.owners_ids)):
```

Reply:
will update